app/scripts/components/exploration/components/datasets/tile-link-modal.tsx

```diff
@@ -29,8 +34,29 @@ export function TileUrlModal(props: {
   tileUrls?: Record<string, string>;
   revealed: boolean;
   onClose: () => void;
+  jupyterUrl: string;
 }) {
-  const { datasetName, tileUrls, revealed, onClose } = props;
+  const { datasetName, tileUrls, revealed, onClose, jupyterUrl } = props;
+
+  const generateQgisUrl = (tileUrl: string) => {
+    const currentDate = new Date().toISOString().split('T')[0];
+    const siteTitle = document.title;
+    const layerName = `${datasetName} at ${currentDate}`;
+    const quotedTileUrl = encodeURIComponent(tileUrl);
```

Comment:
@dzole0311 The tile url needs to be encoded a bit differently. We need to encode the query parameter but leave the rest of the url alone. Here's an example implementation: https://gist.github.com/sunu/06027b5d3f9b86de8027b2be26e45a53#file-index-html-L132-L151